README.md

```diff
@@ -4,6 +4,43 @@
 
 Discord Rich Presence integration for the C* Music Player (`cmus`).
 
+## Installing
+
+- If it isn't already on your system, install `rust`, `cargo`, and `git`. On \*nix based systems this should be as simple as installing it from your package manager.
+
+- Next, clone the git repository with `git clone https://github.com/Bond-009/cmus-discord-rpc`.
```

Comment:
Don't give a specific link, since the project can move and cloning with git is also an option.